packages/sui-jest/src/config/jest.config.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+// TODO: Decide which is the default common config
+const jestConfig = {
+  // All imported modules in your tests should be mocked automatically
```

Comment:
We should remove them if they are not used. We can always read the official docs for the available api.